oscarlab/graphene
# [LibOS] test/apps: Move benchmark-http.sh to common_tools

## Affected components


## Description of the changes 

This PR concerns web-server examples: Apache, Nginx, and Lighttpd. Previously, benchmark-http.sh bash script was located under lighttpd directory, and Apache and Nginx had symbolic links to it. This is unintuitive. This PR moves benchmark-http.sh to a newly created common_tools directory, and all web-server examples have symbolic links to this new dir.

## How to test this PR? 

Apache, Nginx, Lighttpd tests should still run.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1054)
